### DISCUSSION TITLE
Joining the competition by a team

### DISCUSSION THREAD

**Discussion by amirhnourian** [cyberbotics/wrestling#24] [Q&A] [answered]
Dear Mr. Omichel

I appreciate Webots team efforts in this contest our team learnt a lot about webots even until now. 

I wonder how we can join the competition as a team? should we change our name to our team's name?
Is it possible for my colleagues to join the competition individually in order to test their code on your servers? or we must just join it once with our team name and identity?
Moreover, if we are, for instance, in 10th place in the final competition, is it possible to issue the certificate for all team members? 

Thank you for your fun and instructive competition.

**Comment by omichel** [comment, marked as answer]
Dear @amirhnourian,

You are welcome. Yes, this is possible to participate as a team. You can simply change your name to your team's name.
You should list the team members in the description field, so that they will appear as a tool-tip on your team's name on the webpage.
Members of a team should not be part of any other team and should not compete individually.
In order to perform tests, your team members can install Webots on their local computers and test their contribution locally. Then, your team can decide to commit their contribution to the team's controller.
Any individual suspected to contribute to several entries in the leaderboard will be removed.

Regarding certificates, we will issue official certificates for the 32 best ranked participants, including the names of the all team members, at the end of the competition, e.g., on June 2nd, 2023.

I will clarify all this in the official rules ([README.md](https://github.com/cyberbotics/wrestling#readme)).
